$MirOS: ports/lang/egcs/gcc4.4/patches/patch-gcc_doc_cpp_texi,v 1.1 2009/12/06 15:52:18 tg Exp $
--- gcc/doc/cpp.texi.orig	Tue Aug  5 17:24:14 2008
+++ gcc/doc/cpp.texi	Sun Dec  1 04:12:08 2013
@@ -52,7 +52,7 @@ This manual contains no Invariant Sectio
 @ifinfo
 @dircategory Software development
 @direntry
-* Cpp: (cpp).                  The GNU C preprocessor.
+* Cpp-VSS: (cpp-VSS).          The GNU C preprocessor.
 @end direntry
 @end ifinfo
 
